src/Tokenizer/Tokens.php

```diff
@@ -1207,6 +1207,139 @@ public function isPartialCodeMultiline($start, $end)
         return false;
     }
 
+    public function clearTokenMergeSurroundingWhitespaceRemoveLineIfLeftEmpty($index)
```

Comment:
Wouldn't name `clearTokenAndRemoveLineIfLeftEmpty` be enough?

Reply:
renamed inside the new class